app/models/appointment_summary.rb

```diff
@@ -0,0 +1,30 @@
+class AppointmentSummary
+  include ActiveModel::Model
+
+  attr_accessor :supplementary_benefits, :supplementary_debt, :supplementary_ill_health,
+                :supplementary_defined_benefit_pensions, :appointment_type
+
+  %i(supplementary_benefits supplementary_debt
+     supplementary_ill_health supplementary_defined_benefit_pensions).each do |attribute_writer|
+    define_method("#{attribute_writer}=") do |value|
+      boolean = [true, 'true', '1', 1].member?(value)
+      instance_variable_set("@#{attribute_writer}", boolean)
+    end
+
+    alias_method(:"#{attribute_writer}?", attribute_writer)
```

Comment:
This could be a source of weird bugs. Perhaps we could instead define the predicate while iterating over the symbol list using the `[true, 'true', '1', 1].member?(value)` logic and just refer to that in the following writer method that gets defined?